README.md

```diff
@@ -149,3 +149,11 @@ optional arguments:
 
 ```
 
+## Use on Windows
+
+- To use on Windows, change '%TIMETRACKDIR' to the location of your timetrack folder in both `timetrack_multi_cal.bat` and `timetrack_single_cal.bat`.
+- Add the timetrack folder to the system path environment variables. This allows the use of `timetrack_multi_cal` and `timetrack_single_cal` from anywhere in the command line. On Windows 10, this is done via `Control Panel > System and Security > System > Advanced System Settings > Environment Variables`. Then double click on `Path` in System Variables then the `Browse...` button.
```

Comment:
Thanks for adding these instructions regarding the Windows setup. Could you move them further up to `## Setup` (maybe under a separate `### Windows` section), so they're easier to find?